hw/lint/tools/ascentlint/parse-lint-report.py

```diff
@@ -111,13 +111,14 @@ def main():
                    for_json=True,
                    use_decimal=True)
 
-    # return nonzero status if any warnings or errors are present
-    # lint infos do not count as failures
-    n_errors = len(results["errors"]) + len(results["lint_errors"])
-    n_warnings = len(results["warnings"]) + len(results["lint_warnings"])
-    if n_errors > 0 or n_warnings > 0:
-        log.info("Found %d lint errors and %d lint warnings", n_errors, n_warnings)
-        sys.exit(1)
```

Comment:
Don't we still want to exit with status 1 if `results['errors']` or `results['warnings']` is positive?

Reply:
I removed it since the error status of Dvsim is not determined here, and I sort of wanted to avoid that we categorize the message severities in multiple places...
But I can move it back in if you think it would be good to have.

Reply:
Oh, I see. Yep, fine with me.